contracts/test/token/MockToken.sol

```diff
@@ -116,4 +116,40 @@ contract MockToken is ERC20I, MockTokenConfig {
 
         return true;
     }
+
+    /**
+     * @notice It burns an amount of the token of a given
+     *         account.
+     * @param _account The account whose tokens will be burnt.
+     * @param _value The amount that will be burnt.
+     *
+     * @return success `true` if tokens are burned successfully otherwise false
+     */
+    function burn(address _account, uint256 _value) public returns (bool success) {
```

Comment:
I think that `MockToken` just inherit `ERC20Token`.
I believe that there is a PR for this already. (So no need to handle this in this ticket.)